src/extensions/messages.ts

```diff
@@ -69,6 +69,29 @@ export const infoMessages = defineMessages({
   },
 });
 
+export const notifyMessages = defineMessages({
+  extensionReadyToUse: {
+    defaultMessage: "{name} is ready to be used",
+    id: "p42saN",
+    description: "extensions ready to be use",
+  },
+  extensionInstalled: {
+    defaultMessage: "Extension installed",
+    description: "extension has been installed",
+    id: "SItkhR",
+  },
+  extensionRemoved: {
+    defaultMessage: "Extension successfully removed",
+    description: "extension has been removed",
+    id: "JbKGHC",
+  },
+  extensionInstallError: {
+    id: "VokWlV",
+    defaultMessage: "Couldn’t Install {name}",
+    description: "extension has not been installed",
+  },
```

Comment:
suggestion: we can add note that this is about extensions _list_

```suggestion
  extensionInstalled: {
    defaultMessage: "Extension installed",
    description: "extensions list has been installed",
    id: "SItkhR",
  },
  extensionRemoved: {
    defaultMessage: "Extension successfully removed",
    description: "extensions list has been removed",
    id: "JbKGHC",
  },
  extensionInstallError: {
    id: "VokWlV",
    defaultMessage: "Couldn’t Install {name}",
    description: "extensions list has not been installed",
  },
```

additional one:
we can rename `{name}` -> `{extensionName}` to make it clear it's about extension